nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/openvino_backend.py

```diff
@@ -70,7 +70,7 @@ def target_point(target_type: TargetType,
     def create_bias_correction_command(node: NNCFNode,
                                        bias_value: np.ndarray,
                                        nncf_graph: NNCFGraph) -> OVBiasCorrectionCommand:
-        return create_bias_correction_command(node, bias_value, nncf_graph)
+        return OVCommandCreator.create_command_to_update_bias(node, bias_value, nncf_graph)
```

Comment:
Why the same backend-specific entities for other backends weren't added?

Reply:
This is out of the scope of this PR. As we discussed with @alexsu52 this class will be added in the next PRs.